[2025-08-07 11:11:47,730] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T11:11:42.486835+00:00 [queued]>
[2025-08-07 11:11:47,754] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T11:11:42.486835+00:00 [queued]>
[2025-08-07 11:11:47,755] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 11:11:47,755] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-07 11:11:47,755] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 11:11:47,789] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): section_2.inner_section_2.task_2> on 2025-08-06 11:11:42.486835+00:00
[2025-08-07 11:11:47,801] {standard_task_runner.py:52} INFO - Started process 394 to run task
[2025-08-07 11:11:47,809] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_task_group', 'section_2.inner_section_2.task_2', 'scheduled__2025-08-06T11:11:42.486835+00:00', '--job-id', '20', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_task_group.py', '--cfg-path', '/tmp/tmp1a0m6eu1', '--error-file', '/tmp/tmpyz22uzp4']
[2025-08-07 11:11:47,814] {standard_task_runner.py:80} INFO - Job 20: Subtask section_2.inner_section_2.task_2
[2025-08-07 11:11:47,945] {task_command.py:369} INFO - Running <TaskInstance: example_task_group.section_2.inner_section_2.task_2 scheduled__2025-08-06T11:11:42.486835+00:00 [running]> on host b22dcc1493a4
[2025-08-07 11:11:48,361] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_task_group
AIRFLOW_CTX_TASK_ID=section_2.inner_section_2.task_2
AIRFLOW_CTX_EXECUTION_DATE=2025-08-06T11:11:42.486835+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-06T11:11:42.486835+00:00
[2025-08-07 11:11:48,364] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-07 11:11:48,365] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2025-08-07 11:11:48,395] {subprocess.py:85} INFO - Output:
[2025-08-07 11:11:48,403] {subprocess.py:92} INFO - 1
[2025-08-07 11:11:48,405] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-07 11:11:48,479] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_task_group, task_id=section_2.inner_section_2.task_2, execution_date=20250806T111142, start_date=20250807T111147, end_date=20250807T111148
[2025-08-07 11:11:48,549] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-07 11:11:48,655] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
